[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"asyncify-boot","plugin":"genesis-post-images","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.q751LtnHTG"}
,
{"message":"Unknown query type: DESC.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC `wp_extern...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1103): wpdb->get_col('DESC `wp_extern...', 0)\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1058): Puvox\\library_wp->db_table_columns('wp_external_url...')\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#6 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#7 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log` made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, Puvox\\library_wp->db_table_columns, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_external_url_as_post_featured_image_thumbnail__errors_log, `FUNCTION_NAME`, longtext</li>\n<li>Executing: ALTER TABLE wp_external_url_as_post_featured_image_thumbnail__errors_log ADD `function_name` text NOT NULL   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1063): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"Unknown query type: DESC.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC `wp_extern...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1103): wpdb->get_col('DESC `wp_extern...', 0)\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1058): Puvox\\library_wp->db_table_columns('wp_external_url...')\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#6 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#7 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log` made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, Puvox\\library_wp->db_table_columns, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_external_url_as_post_featured_image_thumbnail__errors_log, `FUNCTION_NAME`, longtext</li>\n<li>Executing: ALTER TABLE wp_external_url_as_post_featured_image_thumbnail__errors_log ADD `function_name` text NOT NULL   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1063): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_csr_sign(): X.509 Certificate Signing Request cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 99","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:59 UTC] PHP Warning:  openssl_csr_sign(): X.509 Certificate Signing Request cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 99","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_x509_export(): X.509 Certificate cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 100","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:59 UTC] PHP Warning:  openssl_x509_export(): X.509 Certificate cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 100","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_pkey_export(): Cannot get key from parameter 1 in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 101","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:54:59 UTC] PHP Warning:  openssl_pkey_export(): Cannot get key from parameter 1 in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 101","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_csr_sign(): X.509 Certificate Signing Request cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 99","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:01 UTC] PHP Warning:  openssl_csr_sign(): X.509 Certificate Signing Request cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 99","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_x509_export(): X.509 Certificate cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 100","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:01 UTC] PHP Warning:  openssl_x509_export(): X.509 Certificate cannot be retrieved in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 100","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"openssl_pkey_export(): Cannot get key from parameter 1 in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 101","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:01 UTC] PHP Warning:  openssl_pkey_export(): Cannot get key from parameter 1 in /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php on line 101","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT option_value FROM wp_institution_settings WHERE option_name = 'certificate_cert'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT option_value FROM wp_institution_settings WHERE option_name = :param0  | parameters: certificate_cert</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT option_v...')\n#2 /wordpress/wp-content/plugins/ezpz-sp/class/wps_common.class.php(106): wpdb->get_var('SELECT option_v...')\n#3 /wordpress/wp-content/plugins/ezpz-sp/class/wps_saml.class.php(30): EZPZCommon->get_setting('certificate_cer...')\n#4 /wordpress/wp-content/plugins/ezpz-sp/class/wps_auth.class.php(21): EZPZSAML->__construct(Array)\n#5 /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php(176): EZPZAuth->__construct(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ezpz-sp/ezpzsp....')\n#8 /internal/eval.php(20): activate_plugin('ezpz-sp/ezpzsp....')\n#9 {main}</pre>\n for query SELECT option_value FROM wp_institution_settings WHERE option_name = 'certificate_cert' made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/ezpz-sp/ezpzsp.php'), EZPZAuth->__construct, EZPZSAML->__construct, EZPZCommon->get_setting, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT option_value FROM wp_institution_settings WHERE option_name = 'certificate_key'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT option_value FROM wp_institution_settings WHERE option_name = :param0  | parameters: certificate_key</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT option_v...')\n#2 /wordpress/wp-content/plugins/ezpz-sp/class/wps_common.class.php(106): wpdb->get_var('SELECT option_v...')\n#3 /wordpress/wp-content/plugins/ezpz-sp/class/wps_saml.class.php(31): EZPZCommon->get_setting('certificate_key')\n#4 /wordpress/wp-content/plugins/ezpz-sp/class/wps_auth.class.php(21): EZPZSAML->__construct(Array)\n#5 /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php(176): EZPZAuth->__construct(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ezpz-sp/ezpzsp....')\n#8 /internal/eval.php(20): activate_plugin('ezpz-sp/ezpzsp....')\n#9 {main}</pre>\n for query SELECT option_value FROM wp_institution_settings WHERE option_name = 'certificate_key' made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/ezpz-sp/ezpzsp.php'), EZPZAuth->__construct, EZPZSAML->__construct, EZPZCommon->get_setting, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/g/genesis-post-images/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"genesis-post-images","details":"[19-Apr-2025 16:55:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT option_value FROM wp_institution_settings WHERE option_name = 'certificate_rollover'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT option_value FROM wp_institution_settings WHERE option_name = :param0  | parameters: certificate_rollover</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_institution_settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT option_v...')\n#2 /wordpress/wp-content/plugins/ezpz-sp/class/wps_common.class.php(106): wpdb->get_var('SELECT option_v...')\n#3 /wordpress/wp-content/plugins/ezpz-sp/class/wps_saml.class.php(32): EZPZCommon->get_setting('certificate_rol...')\n#4 /wordpress/wp-content/plugins/ezpz-sp/class/wps_auth.class.php(21): EZPZSAML->__construct(Array)\n#5 /wordpress/wp-content/plugins/ezpz-sp/ezpzsp.php(176): EZPZAuth->__construct(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ezpz-sp/ezpzsp....')\n#8 /internal/eval.php(20): activate_plugin('ezpz-sp/ezpzsp....')\n#9 {main}</pre>\n for query SELECT option_value FROM wp_institutio\n","log":"/logs/plugins/g/genesis-post-images/error.json"}
]
